windows/vbs_enable_disable/vbs_disable_test.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Shutdown guest OS to disable VBS on VM
+- include_tasks: ../utils/win_shutdown_restart.yml
+  vars:
+    set_win_power_state: "shutdown"
+
+# Disable VBS on VM
+- include_tasks: ../utils/win_enable_vbs_vm.yml
+  vars:
+    win_enable_vbs: false
+
+# Power on VM
+- include_tasks: ../../common/vm_set_power_state.yml
+  vars:
+    vm_power_state_set: "powered-on"
+
+# Check VM VBS status on VM
+- include_tasks: ../../common/vm_get_vbs_status.yml
+- name: "Check VM VBS status after enable"
+  ansible.builtin.assert:
+    that:
+      - vm_vbs_enabled is defined
+      - not vm_vbs_enabled | bool
+    fail_msg: "VM VBS status is not disabled after disabling it."
+
+# Get VBS status in guest OS
+- include_tasks: ../utils/win_get_vbs_guest.yml
+
```

Comment:
There could be a rare case that the guest IP is changed after VM reset. So, it is better to check or update VM IP before performing operations in guest.

Reply:
Added get VM IP address and update in-memory inventory then after VM power on. Thanks.